pages/services/ember-consulting.vue

```diff
@@ -1,86 +1,212 @@
 <template>
   <div itemscope itemtype="http://schema.org/Service">
-    <HeroBlock hero-classes="ember-consulting">
-      <template #content>
-        <h1 itemprop="name">Ember Consulting</h1>
-        <p>
-          Ship Shape's ambitious team of experts provide the guidance and
-          manpower that helps you build fast and responsive web applications
-          using Ember.js.
-        </p>
-        <div>
-          <nuxt-link class="btn btn-red" to="/contact/">
-            Start consultation
-          </nuxt-link>
-        </div>
-      </template>
-
-      <template #icon>
-        <nuxt-img
-          class="h-auto m-auto max-w-2xl w-full"
-          format="webp"
-          height="858"
-          src="/img/hero-images/ember-consulting.png"
-          width="1089"
-        />
-      </template>
-    </HeroBlock>
+    <div class="waves white-waves sm:pb-20 lg:pb-8">
+      <HeroBlock class="ember-consulting">
```

Comment:
@hannakim91 I would recommend looking closely at the next.js page in the main branch for the format for things. For example, this `HeroBlock` should not be wrapped in a div, and it takes `hero-classes` to apply classes to it.

Reply:
Thanks, didn't realize the difference was `hero-classes` to get styles to apply to that